lib/rules/no-multiple-empty-lines.js

```diff
@@ -113,8 +113,10 @@ module.exports = {
                 });
 
                 // add the notEmpty lines in there with a placeholder
-                notEmpty.forEach(function(x, i) {
-                    trimmedLines[i] = x;
+                notEmpty.forEach(function(nonEmptyLineNumber) {
+
+                    // FIXME: (not-an-aardvark) Use a better strategy after refactoring this rule
+                    trimmedLines[nonEmptyLineNumber] = "non-empty placeholder string";
```

Comment:
:+1: on this. You probably don't need to modify this strategy unless the logic to search trimmedLines changes.